### ISSUE TITLE
Google Tag Manager Event Data per Button

### ISSUE THREAD

**Issue by ljclark** [verbb/formie#973]
### What are you trying to do?

Currently, when adding event data within the UI for buttons, it applies that event to both buttons if you have a 'Next' and 'Previous'. 

Issue comes that if you have a 'submit' button and a 'previous page' button the final step of a multipage form with some event data such as the below, this gets fired when both buttons are clicked, which can lead to some dud data as the 'previous page' button shouldn't be firing event data like that, it should have it's own event data as it's a very different action. 

![SCR-20220706-k06](https://user-images.githubusercontent.com/8060816/177560551-55828ed3-6707-4670-a8b6-c953843893aa.png)
 

### What's your proposed solution?

There could be two potential options:

1. Add in the ability to be able to set specific event data **per** button within the UI, following the same approach that it currently does.

2. Add in the ability to be be able to render twig or dynamic/button variables within the event data fields. I.e pull in the name of the button into one of the data properties so we can have some dynamic data in the events. This is similar to this FR - #888, but wouldn't necessarily solve this particular issue of 'per button control'

**Comment by engram-design** [comment]
For now, we've fixed this to only fire these events when submitting the form. These won't be triggered when hitting "Back" or the new "Save" button.

To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.16",
  "...": "..."
}
```

Then run `composer update`. 

But I'll be keeping this open as I consider another approach to manage per-button.

**Comment by engram-design** [comment]
Fixed in [2.0.0](https://github.com/verbb/formie/releases/tag/2.0.0)
